module.txt

```diff
@@ -34,6 +34,10 @@
             "id": "Dialogs",
             "minVersion": "1.0.0"
         },
+        {
+            "id": "Economy",
+            "minVersion": "1.1.0-SNAPSHOT"
```

Comment:
As @skaldarnar commented in the associated GD PR: we might want to release `Economy` once the PR is merged and bump the version number.